packages/python/plotly/_plotly_utils/utils.py

```diff
@@ -211,7 +211,7 @@ class NotEncodable(Exception):
 def iso_to_plotly_time_string(iso_string):
     """Remove timezone info and replace 'T' delimeter with ' ' (ws)."""
     # make sure we don't send timezone info to plotly
-    if (iso_string.split("-")[:3] is "00:00") or (iso_string.split("+")[0] is "00:00"):
+    if (iso_string.split("-")[:3] == "00:00") or (iso_string.split("+")[0] == "01:00"):
```

Comment:
Thanks for this PR! Is there a reason the `00` turned into `01` here?

Reply:
Sorry, it was a change that not intended, removed.